src/__integrationtests__/Balance.spec.ts

```diff
@@ -65,7 +70,8 @@ describe('when there is a dev chain with a faucet', () => {
     const funny = jest.fn()
     listenToBalanceChanges(ident.address, funny)
     const balanceBefore = await getBalance(faucet.address)
-    await makeTransfer(faucet, ident.address, MIN_TRANSACTION)
+    const tx = await makeTransfer(faucet, ident.address, MIN_TRANSACTION)
+    await submitTx(tx, AWAIT_IN_BLOCK)
```

Comment:
You have gone down a different route with the following `makeTransfer` see line 99.

Is the `const = tx` needed. Else, it would be better if you make it to a `then()`.

Reply:
it is not, it is just a matter of style. couldn't really decide what I like better